README.md

```diff
@@ -2,42 +2,160 @@
 [![Health](https://landscape.io/github/behdad/fonttools/master/landscape.svg?style=flat)](https://landscape.io/github/behdad/fonttools/master)
 [![Coverage Status](https://img.shields.io/coveralls/behdad/fonttools.svg)](https://coveralls.io/r/behdad/fonttools)
 
-### What it is ?
+### What it this?
 
-Quoting from [TTX/FontTools Sourceforge Project](http://sourceforge.net/projects/fonttools/) 
-> a tool to convert OpenType and TrueType fonts to and from XML. FontTools is a library for manipulating fonts, written in Python. It supports TrueType, OpenType, AFM and to an extent Type 1 and some Mac-specific formats.   
-
-### Quick start
-
-```python setup.py install```
-
-From your command line type the above command to get fontools installed on your system.  FontTools requires Python 2.7, or Python 3.3 or later.
+fontTools is a library for manipulating fonts, written in Python. 
+The project includes the TTX tool, that can convert TrueType and OpenType fonts to and from an XML text format, which is also called TTX.
+It supports TrueType, OpenType, AFM and to an extent Type 1 and some Mac-specific formats.
+The project has a [BSD-style open-source licence](LICENSE.txt).  
+Among other things this means you can use it free of charge. 
 
 ### Installation
 
-See [install.txt](https://github.com/behdad/fonttools/blob/master/Doc/install.txt) in the 'Doc' subdirectory for instructions on how to build and install TTX/FontTools from the sources.
-
-
-### Documentation
-
-#### What is TTX ?
+To download and install fontools on your system, run the following commands:
+FontTools requires Python 2.7, or Python 3.3 or later.
 
-See [documentation.html](https://rawgit.com/behdad/fonttools/master/Doc/documentation.html) in the "Doc" subdirectory for TTX usage instructions and information about the TTX file format.
+```sh
+git clone https://github.com/behdad/fonttools.git ;
+cd fonttools ;
+python setup.py install ;
+```
 
-#### History
+Full build and installation instructions are in [Doc/install.txt](https://github.com/behdad/fonttools/blob/master/Doc/install.txt) 
 
-The fontTools project was started by Just van Rossum in 1999, and was maintained as an open source project at <http://sourceforge.net/projects/fonttools/>. In 2008, Paul Wise (pabs3) began helping Just with stability maintenance. In 2013 Behdad Esfahbod began a friendly fork, thoroughly reviewing the codebase and making changes at <https://github.com/behdad/fonttools> to add new features and support for new font formats. 
```

Comment:
Oops this got lost in the shuffle! I didn't mean to drop it, and I didn't edit it in my drafting either (other than to put one sentence per line)